nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/core/ElasticSearchIndexingImpl.java

```diff
@@ -178,6 +180,12 @@ void processBulkIndexCommands(List<IndexingCommand> cmds) {
                     bulkSize += idxRequest.source().length();
                     bulkRequest.add(idxRequest);
                 }
+            } catch (PropertyException pe) {
+                if (pe.getMessage().contains(CANNOT_GET_BLOB_INFO)) {
+                    log.info("Ignoring missing blob info for " + cmd);
```

Comment:
Don't log as `INFO`, the guy reading the log files won't be able to do anything about it.

Reply:
The code previously threw an exception with stack trace.  If they have lots of missing blobs then that indicates a problem.  I could change it to log.debug but assume that would never get turned on.  So essentially I would be suppressing any knowledge of the issue.  Is that your preference?

Reply:
In this light `INFO` is ok